servers/rendering/renderer_rd/shaders/effects/copy.glsl

```diff
@@ -103,15 +103,16 @@ void main() {
 #ifdef MODE_GLOW
 	if (bool(params.flags & FLAG_GLOW_FIRST_PASS)) {
 		// Tonemap initial samples to reduce weight of fireflies: https://graphicrants.blogspot.com/2013/12/tone-mapping.html
-		local_cache[dest_index] /= 1.0 + dot(local_cache[dest_index].rgb, vec3(0.299, 0.587, 0.114));
-		local_cache[dest_index + 1] /= 1.0 + dot(local_cache[dest_index + 1].rgb, vec3(0.299, 0.587, 0.114));
-		local_cache[dest_index + 16] /= 1.0 + dot(local_cache[dest_index + 16].rgb, vec3(0.299, 0.587, 0.114));
-		local_cache[dest_index + 16 + 1] /= 1.0 + dot(local_cache[dest_index + 16 + 1].rgb, vec3(0.299, 0.587, 0.114));
+		vec3 tonemap_col = vec3(0.299, 0.587, 0.114) / max(params.glow_luminance_cap, 6.0);
+		local_cache[dest_index] /= 1.0 + dot(local_cache[dest_index].rgb, tonemap_col);
+		local_cache[dest_index + 1] /= 1.0 + dot(local_cache[dest_index + 1].rgb, tonemap_col);
+		local_cache[dest_index + 16] /= 1.0 + dot(local_cache[dest_index + 16].rgb, tonemap_col);
+		local_cache[dest_index + 16 + 1] /= 1.0 + dot(local_cache[dest_index + 16 + 1].rgb, tonemap_col);
 	}
-	const float kernel[4] = { 0.174938, 0.165569, 0.140367, 0.106595 };
+	const float kernel[5] = { 0.2024, 0.1790, 0.1240, 0.0672, 0.0285 };
 #else
 	// Simpler blur uses SIGMA2 for the gaussian kernel for a stronger effect.
-	const float kernel[4] = { 0.214607, 0.189879, 0.131514, 0.071303 };
+	const float kernel[5] = { 0.214607, 0.189879, 0.131514, 0.071303, 0.0285 };
```

Comment:
Just flagging that the last value here seems to be unused in this branch. I guess it's better to have an unused value than having `kernel` size differ for glow and non-glow?

Reply:
This line can just be reverted. The 5th index is only ever accessed when using MODE_GLOW

Reply:
I removed that, thanks for the feedback!